docs/developer-docs/identity/verifiable-credentials/how-it-works.mdx

```diff
@@ -77,6 +77,16 @@ From the developer’s perspective, the interaction between dapps uses window me
 
 For more details, check out the [specification in Internet Identity’s repository](https://github.com/dfinity/internet-identity/blob/main/docs/vc-spec.md).
 
+## Test Verifiable Credentials
+
+There is a dummy relying party deployed in Internet Computer here: [https://l7rua-raaaa-aaaap-ahh6a-cai.icp0.io/](https://l7rua-raaaa-aaaap-ahh6a-cai.icp0.io/).
+
+You first need to login with an Identity Provider. To avoid using real users, we recommend using the staging Internet Identity deployed here: [https://fgte5-ciaaa-aaaad-aaatq-cai.ic0.app/](https://fgte5-ciaaa-aaaad-aaatq-cai.ic0.app/).
+
+We have deployed a dummy issuer that will issue any requested credential. Feel free to use it when testing the flow.
+
+The dummy issuer is deployed in [canister id qdiif-2iaaa-aaaap-ahjaq-cai](https://dashboard.internetcomputer.org/canister/qdiif-2iaaa-aaaap-ahjaq-cai).
```

Comment:
```suggestion
There is a demo issuer deployed on ICP  available for testing. It will issue any requested credential. Feel free to use it when testing the flow: [qdiif-2iaaa-aaaap-ahjaq-cai](https://dashboard.internetcomputer.org/canister/qdiif-2iaaa-aaaap-ahjaq-cai).
```